Makefile

```diff
@@ -162,7 +162,7 @@ commitlint:
 .PHONY: cephcsi
 cephcsi: check-env
 	if [ ! -d ./vendor ]; then (go mod tidy && go mod vendor); fi
-	GOOS=linux go build $(GO_TAGS) -mod vendor -a -ldflags '$(LDFLAGS)' -o _output/cephcsi ./cmd/
+	GOOS=linux go build $(GO_TAGS) -buildvcs=false -mod vendor -a -ldflags '$(LDFLAGS)' -o _output/cephcsi ./cmd/
```

Comment:
this now got added, is that intentional?

Reply:
My bad.  it is not intentional